base/server/python/pki/server/cli/audit.py

```diff
@@ -123,32 +255,46 @@ def execute(self, argv):
                   % (subsystem_name.upper(), instance_name))
             sys.exit(1)
 
+        name = 'log.instance.SignedAudit.%s'
+
         if enabled is None:
             pass
         elif enabled:
-            subsystem.config['log.instance.SignedAudit.enable'] = 'true'
+            subsystem.config[name % 'enable'] = 'true'
         else:
-            subsystem.config['log.instance.SignedAudit.enable'] = 'false'
+            subsystem.config[name % 'enable'] = 'false'
+
+        if logFile:
+            subsystem.config[name % 'fileName'] = logFile
+
+        if bufferSize:
+            subsystem.config[name % 'bufferSize'] = bufferSize
+
+        if flushInterval:
+            subsystem.config[name % 'flushInterval'] = flushInterval
+
+        if maxFileSize:
+            subsystem.config[name % 'maxFileSize'] = maxFileSize
+
+        if rolloverInterval:
+            subsystem.config[name % 'rolloverInterval'] = rolloverInterval
+
+        if expirationTime:
+            subsystem.config[name % 'expirationTime'] = expirationTime
 
         if logSigning is None:
             pass
         elif logSigning:
-            subsystem.config['log.instance.SignedAudit.logSigning'] = 'true'
+            subsystem.config[name % 'logSigning'] = 'true'
         else:
-            subsystem.config['log.instance.SignedAudit.logSigning'] = 'false'
+            subsystem.config[name % 'logSigning'] = 'false'
 
-        if maxFileSize:
-            subsystem.config['log.instance.SignedAudit.maxFileSize'] = maxFileSize
```

Comment:
Should we be removing the `maxFileSize` from print output? I see that you try to print `        maxFileSize = subsystem.config[name % 'maxFileSize']` in line 61

Reply:
It's actually moved to line 276 in the new code. This code basically will update max file size only if the --maxFileSize param is specified. We'll always display the max file size in line 61 & 74.